c_glib/arrow-glib/type.h

```diff
@@ -72,6 +74,7 @@ typedef enum {
   GARROW_TYPE_DOUBLE,
   GARROW_TYPE_STRING,
   GARROW_TYPE_BINARY,
+  GARROW_TYPE_FIXED_SIZE_BINARY,
```

Comment:
Putting a new enum value into middle of enums breaks a ABI compatibility.
But we don't need to care for now. Because we bump shared library version for each version. We can't do this when we release 1.0.0.